mods/ts/sequences/infantry.yaml

```diff
@@ -1483,61 +1302,70 @@ doggie: # TODO: not sure what frame 88 and following is
 		Start: 8
 		Length: 6
 		Facings: 8
-		Tick: 80
 		ShadowStart: 127
+	shoot:
+		Start: 56
+		Length: 4
+		Facings: 8
+		ShadowStart: 175
+	laydown: # TODO: Implement hiding when on a tiberium patch
+		Start: 88
+		Length: 2
+		ShadowStart: 207
+	hide: # TODO: Implement hiding when on a tiberium patch
```

Comment:
I'm pretty sure 'hiding' was just a visual bonus as they looked like the Tiberium.
They weren't invisible.